File: Assets/VirtualCity/ProtoDefine/ReqQueryCommentsMessage.cs
using System.Collections.Generic;
using ProtoBuf;
namespace ProtoDefine {
[ProtoContract]


public class ReqQueryCommentsMessage {
    /**星数*/
[ProtoMember(1)]
    public int star;
    /**最后一条记录的时间*/
[ProtoMember(2)]
    public string lastDate;
    /**商品Id*/
[ProtoMember(3)]
    public long goodsId;

    public int getStar() {
        return star;
    }

    public void setStar(int star) {
        this.star = star;
    }

    public string getLastDate() {
        return lastDate;
    }

    public void setLastDate(string lastDate) {
        this.lastDate = lastDate;
    }

    public long getGoodsId() {
        return goodsId;
    }

    public void setGoodsId(long goodsId) {
        this.goodsId = goodsId;
    }
}
}

File: Assets/VirtualCity/ProtoDefine/Java/mall/ReqQueryCommentsMessage.java
package com.kingston.jforgame.server.game.mall.message;

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;
import com.kingston.jforgame.server.game.Modules;
import com.kingston.jforgame.server.game.mall.MallDataPool;
import com.kingston.jforgame.socket.annotation.MessageMeta;
import com.kingston.jforgame.socket.message.Message;

@MessageMeta(module = Modules.SHOPPING,cmd = MallDataPool.REQ_QUERYCOMMENT)
public class ReqQueryCommentsMessage extends Message {
    /**星数*/
    @Protobuf(order = 1)
    private int star;
    /**最后一条记录的时间*/
    @Protobuf(order = 3)
    private String lastDate;
    /**商品Id*/
    @Protobuf(order = 4)
    private long goodsId;

    public int getStar() {
        return star;
    }

    public void setStar(int star) {
        this.star = star;
    }

    public String getLastDate() {
        return lastDate;
    }

    public void setLastDate(String lastDate) {
        this.lastDate = lastDate;
    }

    public long getGoodsId() {
        return goodsId;
    }

    public void setGoodsId(long goodsId) {
        this.goodsId = goodsId;
    }
}
